src/lib/dfp/should-refresh.ts

```diff
@@ -24,6 +24,11 @@ const shouldRefresh = (
 ): boolean => {
 	const sizeString = advert.size?.toString();
 
+	// Do not refresh ads in slots labelled data-refresh="false"
+	if (advert.node.dataset.refresh === 'false') {
+		return false;
+	}
```

Comment:
I've moved this condition that handles `data-refresh="true"` in here to ensure all of the code that decides whether a given advert should refreshed is located in a single place.